prost-build/src/lib.rs

```diff
@@ -490,7 +490,7 @@ impl Config {
     ///
     /// ```rust,ignore
```

Comment:
I wonder if we want `no_run` here instead so we catch this on CI?

Reply:
Thanks for the review. Fixed to use no_run instead of ignore. 

https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes